components/Wallet/Message/Detail/index.jsx

```diff
@@ -4,8 +4,10 @@ import { FilecoinNumber } from '@glif/filecoin-number'
 import { func } from 'prop-types'
 import styled from 'styled-components'
 import dayjs from 'dayjs'
-
+import { speedUpTransaction } from '../../../../utils/modifyTransaction'
+import { useRouter } from 'next/router'
```

Comment:
tiny thing that the linter should catch, but lets put all npm imports above relative ones